src/custom_nodes/horizontal_ocr/config.json

```diff
@@ -57,7 +57,8 @@
                         "convert_to_gray_scale": "true",
                         "confidence_threshold": "0.3",
                         "max_output_batch": "100",
-                        "debug": "true"
+                        "debug": "true",
+                        "buffer_queue_size": "4"
```

Comment:
Is this optimal default size?

Reply:
horizontal_ocr CN is used in camera_client. By default that client uses 4 threads for parallel service requesting, therefore I thouht that this size would be optimal for the example. With greater number of threads it will not be optimal tho.

Reply:
@mkuczyns11 what if the node extracts more than 1 horizontal text on the image? This would be `no_of_clients * no_of_horizontal_texts_on_image` buffers needed

Reply:
No, `no_of_clients` buffers are still needed. The reason is that one buffer has size equal to `max_output_batch * size_of_one_image`. It means that no matter how many detections may happen, we will be able to write them into the buffer(that is since before saving into outputs we only select max_output_batch or less amount of detections by their score).

Reply:
I decided to change default queue_size to 24. It should not have an impact on execution speed and it enables possibility to use more clients(only initialization should be longer).